### DISCUSSION TITLE
Forms and multi-page forms

### DISCUSSION THREAD

**Discussion by supermx1** [Budibase/budibase#1062] [Help]
Hello, I was thinking about how to create a multipage form? Is this an idea that can be implemented easily?

**Comment by joebudi** [comment]
Hi. We are working on improving the process for creating forms within the builder. You will not be able to create a multipage form. If we created a tab component, that would allow you to break the form up into different sections. Would that suffice?

**Reply by supermx1** [reply]
Well, that would suffice. Is that feature available now or in a future release?

**Reply by marcosfelt** [reply]
Any news on this?

**Reply by joebudi** [reply]
Hey!

We've scheduled work in August for this:
https://github.com/Budibase/budibase/issues/1765

So, it's not available now, but it's coming soon (Aug release).

**Reply by marcosfelt** [reply]
Sounds great! :)

**Reply by mgzon** [reply]
Hi,
Any update on this tab component.
